marklogic-unit-test-modules/src/main/ml-modules/root/test/test-helper.xqy

```diff
@@ -771,14 +830,14 @@ declare function test:assert-meets-minimum-threshold(
         $message,
         "actual: " || xdmp:describe($actual) || " is less than minimum: " || xdmp:describe($minimum)
```

Comment:
Mind changing the error message here to more closely align with the function name?

Instead of:

`"actual: " || xdmp:describe($actual) || " is less than minimum: " || xdmp:describe($minimum)`

We should say:

`xdmp:describe($actual) || " is not greater than or equal to  " || xdmp:describe($minimum)`

Reply:
Done.